docs/serving/samples/cloudevents/cloudevents-spring/README.md

```diff
@@ -0,0 +1,139 @@
+# Spring + CloudEvents + Knative
+
+A simple web app written in Java using Spring Cloud Function that can receive CloudEvents. It
+supports running in two modes:
+
+1. The default mode has the app reply to your input events with the output
+   event, which is simplest for demonstrating things working in isolation, but
+   is also the model for working for the Knative Eventing `Broker` concept. The
+   input event is modified assigning a new source and type attribute.
+
+2. `K_SINK` mode has the app send events to the destination encoded in
+   `$K_SINK`, which is useful to demonstrate how folks can synthesize events to
+   send to a Service or Broker when not initiated by a Broker invocation (e.g.
+   implementing an event source). The input event is modified assigning a new
+   source and type attribute.
+
+The application will use `$K_SINK`-mode whenever the environment variable is
```

Comment:
nit for consistency, maybe drop the $ since above you say K_SINK mode?

Reply:
Copy paste. If you want to fix this sample, you would want to fix the other(s). Can we make that a separate patch?